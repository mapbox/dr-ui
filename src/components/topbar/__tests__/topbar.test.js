import renderer from 'react-test-renderer';
import { testCases } from './topbar-test-cases.js';

describe('topbar-sticker', () => {
  describe(testCases.basic.description, () => {
    let testCase;
    let wrapper;
    let tree;

    beforeEach(() => {
      testCase = testCases.basic;
      wrapper = renderer.create(testCase.element);
      tree = wrapper.toJSON();
    });

    test('renders as expected', () => {
      expect(tree).toMatchSnapshot();
    });
  });
});
