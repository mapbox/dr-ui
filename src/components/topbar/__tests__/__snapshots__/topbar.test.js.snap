// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`topbar-sticker Basic renders as expected 1`] = `
<div
  className="border-t border-b border--gray-light bg-white"
  data-swiftype-index="false"
>
  <div
    className="limiter"
  >
    <div
      className="grid"
    >
      <div
        className="col col--4-mm col--12"
      >
        <div
          className="ml24-mm pt12"
          style={
            Object {
              "height": 52,
            }
          }
        >
          <div
            className="dr-ui--product-menu"
          >
            <a
              className="txt-fancy txt-l block color-blue-on-hover"
              href="/dr-ui/"
            >
              Dr. UI
            </a>
          </div>
        </div>
      </div>
      <div
        className="col col--8-mm col--12"
      >
        <div
          className="flex-parent-mm flex-parent--center-cross flex-parent--end-main h-full-mm wmax300 wmax-full-mm mb0-mm mb12"
        >
          <div
            className="h36 relative"
          >
            <div>
              <div>
                <button
                  className="flex-parent flex-parent--center-cross btn--gray color-gray-light btn btn--stroke py3 pl6 pr12 round w-full"
                  onClick={[Function]}
                  style={Object {}}
                >
                  <span
                    className="mr6 color-gray"
                  >
                    <svg
                      className="icon w18 h18"
                    >
                      <use
                        xlinkHref="#icon-search"
                      />
                    </svg>
                  </span>
                   
                  <span
                    className="color-gray"
                  >
                    Search
                  </span>
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
